pkg/apis/jaegertracing/v1/options.go

```diff
@@ -7,10 +7,28 @@ import (
 	"strings"
 )
 
+// Values represent and string or a slice of string
```

Comment:
This doesn't seem accurate. Perhaps you meant "Values hold a map, with string as the key and either a string or a slice of strings as the value"

In any case, the name for this struct isn't really telling what the struct is. Can you come up with a better name?

Reply:
Better comment :) I'll change it, and rename this to a better name